pyproject.toml

```diff
@@ -43,10 +46,20 @@ line-length = 120
 exclude_lines = [
     "raise NotImplementedError",
     "if TYPE_CHECKING",
+    "if __name__",
+    "async def main",
     "pragma: no cover",
+    "import",
+    "Settings",
```

Comment:
Why?

Reply:
coverage is not ignoring imports and i don't think there should be tests for Settings instance